MANIFEST.in

```diff
@@ -1,4 +1,5 @@
 include setup.py src/sentry/assets.json README.rst MANIFEST.in LICENSE AUTHORS
+recursive-include ./ requirements*.txt
```

Comment:
I mean, I really don't think it'd be good to just add `./` but I'm not 100% sure what this will do without testing it manually. Why not just `requirements-*.txt` ?

Reply:
@mattrobenolt thats what this is doing as far as i understand

Reply:
Oh, becasue within `recursive-include`. Yeah, might be fine? I'll check it manually after merging.